info.xml

```diff
@@ -23,8 +23,11 @@
     to provide richer email features.
   </comments>
   <compatibility>
-    <ver>5.13</ver>
+    <ver>5.19</ver>
   </compatibility>
+  <classloader>
+    <psr4 prefix="Civi\" path="Civi"/>
+  </classloader>
```

Comment:
Huzza for -27 SLOC on `_flexmailer_autoload()`!

To replace it, did you try just:

```xml
 <psr4 prefix="Civi\FlexMailer\" path="src"/>
```

Then it's not necessary to rename the files. The benefit of PSR-4 (versus the older PSR-0) is that it allows cleaner/less-redundant file-trees.

Reply:
@totten This took a while for me to circle back around to! But you are right it works so I've updated the PR